overlord/snapstate/autorefresh.go

```diff
@@ -59,11 +59,12 @@ func newAutoRefresh(st *state.State) *autoRefresh {
 	}
 }
 
-// RefreshSchedule will return a user visible string with the current
-// schedule for the automatic refreshes.
-func (m *autoRefresh) RefreshSchedule() (string, error) {
-	_, scheduleStr, err := m.refreshScheduleWithDefaultsFallback()
-	return scheduleStr, err
+// RefreshSchedule will return a user visible string with the current schedule
+// for the automatic refreshes and a flag indicating whether the schedule is a
+// legacy one.
+func (m *autoRefresh) RefreshSchedule() (string, bool, error) {
```

Comment:
I would name the return values now to make clear what the "bool" stands for, something like `func ... (schedule string, legacy bool, err error)`